.github/workflows/validate-api.yml

```
@@ -0,0 +1,105 @@
+    
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+    
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install fastapi uvicorn pydantic pydantic-settings PyYAML
+        pip install openapi-spec-validator
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)